src/main/java/org/kuali/kfs/integration/cg/ContractsAndGrantsBillingAward.java

```diff
@@ -0,0 +1,381 @@
+/**
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ *
+ * Copyright 2005-2019 Kuali, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.kuali.kfs.integration.cg;
+
+import org.kuali.kfs.integration.ar.AccountsReceivableBillingFrequency;
+import org.kuali.kfs.integration.ar.Billable;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.kim.api.identity.Person;
+
+import java.sql.Date;
+import java.sql.Timestamp;
+import java.util.List;
+
+/**
+ * Integration interface for Award (specific to CGB functionality)
+ */
+public interface ContractsAndGrantsBillingAward extends Billable, ContractsAndGrantsAward {
```

Comment:
Can we add a comment to the function that was added or updated?  I am not sure what our customization is